overlord/snapstate/export_test.go

```diff
@@ -149,20 +149,20 @@ func ByKindOrder(snaps ...*snap.Info) []*snap.Info {
 }
 
 func MockModelWithBase(baseName string) (restore func()) {
-	return mockModel(baseName)
+	return mockModel(fmt.Sprintf("\nbase: %s", baseName))
+}
+
+func MockModelWithKernelTrack(kernelTrack string) (restore func()) {
+	return mockModel(fmt.Sprintf("\nkernel-track: %s", kernelTrack))
 }
 
 func MockModel() (restore func()) {
 	return mockModel("")
 }
 
-func mockModel(baseName string) (restore func()) {
+func mockModel(extra string) (restore func()) {
```

Comment:
Ah, this explains my questions above.

How about making any non-empty value of `extra` also add the newline automatically. It's also less error prone to use.